app/controllers/welcome_controller.rb

```diff
@@ -1,3 +1,5 @@
 class WelcomeController < ApplicationController
-  def index; end
+  def index
+    render file: 'welcome/index.html'
```

Comment:
why do you need this?

Reply:
Without this heroku fails, seems that's why i've added it here.